pallets/inflation/src/lib.rs

```diff
@@ -0,0 +1,256 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+#[cfg(test)]
+mod tests;
+use frame_support::{
+	dispatch::{DispatchResult},
+	traits::{Currency, Get},
+};
+pub use pallet::*;
+use sp_runtime::{
+	Perbill,
+	traits::{BlockNumberProvider},
+};
+
+type BalanceOf<T> =
+<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
+
+pub const YEAR: u32 = 2_629_800; // 5_259_600 6-second block
+// pub const YEAR: u32 = ; // 12-second block
```

Comment:
Can we setup here 12 second block because that how it is on our parachain

Reply:
So 12 seconds block would be 2_628_000? I am not sure why they put 2_629_800